tests/unit/test_charm.py

```diff
@@ -9,12 +9,13 @@
 
 import pytest
 import yaml
-from charm import KafkaK8sCharm
-from literals import CHARM_KEY, CONTAINER, PEER, REL_NAME, STORAGE, ZOOKEEPER_REL_NAME
 from ops.model import BlockedStatus, WaitingStatus
 from ops.testing import Harness
 from tenacity.wait import wait_none
 
+from charm import KafkaK8sCharm
+from literals import CHARM_KEY, CONTAINER, PEER, REL_NAME, STORAGE, ZOOKEEPER_REL_NAME
+
```

Comment:
**praise:** Thanks for this, much nicer looking!

Reply:
you should really thanks black :P 

I did not do much here